datastacks_tests/unit/test_utils.py

```diff
@@ -38,34 +39,138 @@ def test_render_template_components(tmp_path):
         assert Path(f"{target_dir}/{file_path}").exists()
 
 
+@patch("datastacks.utils.click.confirm")
 @patch("datastacks.utils.generate_target_dir")
-def test_generate_pipeline_no_dq(mock_target_dir, tmp_path):
+def test_generate_pipeline_no_dq(mock_target_dir, mock_confirm, tmp_path):
     mock_target_dir.return_value = tmp_path
+    mock_confirm.return_value = "Y"
     config_path = "datastacks_tests/unit/test_config.yml"
     template_source_folder = INGEST_TEMPLATE_FOLDER
 
-    target_dir = generate_pipeline(config_path, False, template_source_folder, "ingest")
+    target_dir = generate_pipeline(config_path, False, template_source_folder, "Ingest")
```

Comment:
I guess this "Ingest" could be a const prefix set in the datastacks package. Feel free to update or ignore & resolve.

Reply:
Gonna leave as is for now but it's a fair suggestion